.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -23,12 +23,8 @@ Submariner Owners: https://github.com/orgs/submariner-io/teams/submariner-core
 **Anything else we need to know?**:
 
 **Environment**:
-- Submariner version (use `subctl version`):
-- Kubernetes version (use `kubectl version`):
 - Diagnose information (use `subctl diagnose all`):
 - Gather information (use `subctl gather`)
 - Cloud provider or hardware configuration:
-- OS (e.g `cat /etc/os-release`):
```

Comment:
Gather collects these from the host, not just containers?

Reply:
@dfarrell07 Yes. From each node by parsing `Node.Status.NodeInfo`
https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/gather/summary.go#L154-L167

and the output looks like this (not properly formatted here due to copy paste)

```
Node name | Operating System | Container Runtime Version | Kubelet Version | KubeProxy Version
ip-10-0-129-36.ec2.internal | linux Red Hat Enterprise Linux CoreOS 47.83.202106252242-0 (Ootpa) 4.18.0-240.22.1.el8_3.x86_64 amd64 | cri-o://1.20.3-6.rhaos4.7.git0d0f863.el8 | v1.20.0+87cc9a4 | v1.20.0+87cc9a4
```

Reply:
Right, so that's the node OS/similar. I understood the Issue Template to be asking for the host OS/similar, like the machine you're running `subctl` from.

Reply:
Sure @dfarrell07 Also if you can vote on this issue and merge it please.

Reply:
I think we merged this without addressing my -1.

We shouldn't dismiss reviews unless the feedback is clearly fixed.